lib/alchemy/sass-support.rb

```diff
@@ -0,0 +1,11 @@
+module Alchemy
+  begin
+    require 'sassc-rails'
+  rescue LoadError, Gem::LoadError => e
+    begin
+      require 'sass-rails'
+    rescue LoadError, Gem::LoadError => e
+      raise LoadError, "Could not find the `sass-rails` or `sassc-rails` gem! Please add one of them to your project's Gemfile."
```

Comment:
Should we add " for AlchemyCMS" into this error message, so users can see more easily where this is coming from?